docs/inputs.md

```diff
@@ -1260,6 +1267,167 @@ regional_pass: 10.90
     </tr>
 </table>
 
+<a id="cvm_establishment_synthesis"></a>
+
+### CVM Establishment Synthesis
+#### `PERCENT_OF_ESTABLISHMENTS_BY_LUZ_SIZE_EMP_CAT.CSV`
+
+<table>
+    <tr>
+        <th>Column Name</th>
+        <th>Description</th>
+    </tr>
+    <tr>
+        <td>Info</td>
+        <td>
+            Basic information of the data, and the size class definitions included
+        </td>
+    </tr>
+    <tr>
+        <td>Data</td>
+        <td>Detailed industry sector breakdown by size class and LUZ</td>
+    </tr>
+    
+</table>
+
+<a id ="cvm"></a>
+
+### Commercial Vehicle Model ( CVM )
+#### `SYNTHESTABLISHMENTS.CSV`
```

Comment:
Where are the other CVM input files? "MGRAEmpByEstSize.csv", "SummarySynthEstabs.csv"